templates/dockerignore

```diff
@@ -1,5 +1,12 @@
-fly.toml
-Dockerfile
-.dockerignore
-node_modules
 .git
+node_modules
+.dockerignore
+.env
+Dockerfile
+fly.toml
+<% if (remix) { -%>
```

Comment:
This would work better if `.dockerignore` was an ejs file.   See https://github.com/fly-apps/dockerfile-node/blob/995ff6da23a3e8df1ca71d41bffefae93a637b2f/gdf.js#L222-L225 .  A few lines above this is code that processes Dockerfile.ejs as a template, that code could be adapted.

Reply:
I _think_ I implemented what you asked.
Can't check as I'm not behind my computer anymore atm